navbar.webpack.config.coffee

```diff
@@ -0,0 +1,20 @@
+webpackConfig = require 'appirio-tech-webpack-config'
+webpack = require('webpack')
+
+config = webpackConfig
+  dirname: __dirname
+  template: './index.html'
+  entry:
+    TCNavComponents: [
+      './index.coffee'
+    ]
+
+config.output.filename = 'TCNavComponents.[name].js';
+config.output.publicPath = 'http://local.topcoder.com/mf/js/app/header/partials/'
+config.output.libraryTarget = 'var'
```

Comment:
Yes, it would bundle it as library to be used in a script tag. It has different values for different usage e.g. using in "commonjs", "amd" etc and default to 'var'. I think I can remove it because it is already a default value.
Anyway, it seems two different set of changes are merged into one PR, I would separate out changes for SubNav and bundling the NavBar to separate PRs for better tracking.